.circleci/config.yml

```diff
@@ -108,12 +114,24 @@ jobs:
     - run:
         name: Lint JS
         command: npm run lint:js
-    - run:
-        name: Lint CSS Size
-        command: npm run lint:size
+        when: always
     - run:
         name: Lint CSS
         command: npm run lint:css
+        when: always
+    - restore_cache:
```

Comment:
No need to restore_cache

Reply:
done

Reply:
I removed restore cache for the below. You noted above the 'Lint CSS' target.
    - run:
        name: Lint CSS Size
        command: .circleci/css-size-report/css-size-report.sh
        when: always